services/web/src/modals/InviteUser.js

```diff
@@ -13,8 +13,30 @@ export default class InviteUser extends React.Component {
     touched: false,
     validEmails: [],
     invalidEmails: [],
+    roles: [],
   };
 
+  componentDidMount() {
+    this.fetchRoles();
+  }
+
+  async fetchRoles() {
+    const { data } = await request({
+      method: 'GET',
+      path: `/1/users/roles`,
+    });
+    const roles = Object.keys(data).map((key) => {
```

Comment:
small nit but I kind of prefer setting state data as-is from the server and transforming it for the sake of dropdown options when rendering instead

Reply:
I don't see how. I mean we use this pattern everywhere and in either case it's a single object set on the state when the data is loaded... transformed or not it never changes